src/pages/gen2/start/quickstart/nextjs-app-router-server-components/index.mdx

```diff
@@ -0,0 +1,351 @@
+export const meta = {
+  title: 'Next.js App Router (Server Components)',
+  description: 'Get started with AWS Amplify (Gen 2) using the Next.js App Router using Server Components.'
+};
+
+export function getStaticProps(context) {
+  return {
+    props: {
+      meta
+    }
+  };
+}
+
+This Quickstart guide will walk you through how to build a task list application with TypeScript, Next.js **App Router with Server Components**, and React. If you are new to these technologies, we recommend you go through the official [React](https://react.dev/learn/tutorial-tic-tac-toe), [Next.js](https://nextjs.org/docs/getting-started/installation), and [TypeScript](https://www.typescriptlang.org/docs/handbook/typescript-from-scratch.html) tutorials first.
+
+import prerequisites from 'src/fragments/gen2/quickstart/prerequisites.mdx';
+
+<Fragments fragments={{ javascript: prerequisites, nextjs: prerequisites }} />
+
+import createProject from 'src/fragments/gen2/quickstart/create-nextjs-app-router-project.mdx';
+
+<Fragments fragments={{ javascript: createProject, nextjs: createProject }} />
+
+import buildABackend from 'src/fragments/gen2/quickstart/build-a-backend.mdx';
+
+<Fragments fragments={{ javascript: buildABackend, nextjs: buildABackend }} />
+
+
+## Build UI
+
+Let's add UI that connects to the backend data and auth resources.
+
+
+### Configure Amplify Client Side
+
+First, install the Amplify UI component library:
+
+```bash
+npm install @aws-amplify/ui-react
+```
+
+Next, create a `components` folder in the root of your project and the contents below to a file called `ConfigureAmplify.tsx`.
+
+```ts title="components/ConfigureAmplify.tsx"
+// components/ConfigureAmplify.tsx
+"use client";
+
+import { Amplify } from "aws-amplify";
+
+import config from "@/amplifyconfiguration.json";
+
+Amplify.configure(config, { ssr: true });
+
+export default function ConfigureAmplifyClientSide() {
+  return null;
+}
+```
+
+Update `app/layout.tsx` to import and render `<ConfigureAmplifyClientSide />`.  This client component will configure Amplify for client pages in our application.
+
+```ts title="app/layout.tsx"
+// app/layout.tsx
+import "@aws-amplify/ui-react/styles.css";
+import type { Metadata } from "next";
+import { Inter } from "next/font/google";
+import "./globals.css";
+
+import ConfigureAmplifyClientSide from "@/components/ConfigureAmplify";
+
+const inter = Inter({ subsets: ["latin"] });
+
+export const metadata: Metadata = {
+  title: "Create Next App",
+  description: "Generated by create next app",
+};
+
+export default function RootLayout({
+  children,
+}: {
+  children: React.ReactNode;
+}) {
+  return (
+    <html lang="en">
+      <body className={inter.className}>
+        <ConfigureAmplifyClientSide />
+        {children}
+      </body>
+    </html>
+  );
+}
+```
+
+### Add a login page
+
```

Comment:
@kevinold It would be good to have a sentence here describing this box and prompting to create this file since it will be new